src/blockchain/electrum.rs

```diff
@@ -47,9 +47,11 @@ pub struct ElectrumBlockchain(Client);
 
 #[cfg(test)]
 #[cfg(feature = "test-electrum")]
-#[bdk_blockchain_tests(crate)]
-fn local_electrs() -> ElectrumBlockchain {
-    ElectrumBlockchain::from(Client::new(&testutils::get_electrum_url()).unwrap())
+testutils::bdk_blockchain_tests! {
+    bdk => crate,
+    fn test_instance() -> ElectrumBlockchain {
+        ElectrumBlockchain::from(Client::new(&testutils::get_electrum_url()).unwrap())
+    }
```

Comment:
Maybe this should just be at the end of the file I see no reason to clutter the start of it up here.